tests/by-util/test_pmap.rs

```diff
@@ -20,6 +20,74 @@ fn test_no_args() {
     new_ucmd!().fails().code_is(1);
 }
 
+#[test]
+#[cfg(target_os = "linux")]
+fn test_default_rc() {
+    let pid = process::id();
+    let ts = TestScenario::new(util_name!());
+
+    // Fails to read before creating rc file
+    for arg in ["-c", "--read-rc"] {
+        ts.ucmd().arg(arg).arg(pid.to_string()).fails().code_is(1);
+    }
+
+    // Create rc file
+    ts.ucmd().arg("-n").succeeds();
+
+    // Fails to create because rc file already exists
+    for arg in ["-n", "--create-rc"] {
+        ts.ucmd().arg(arg).fails().code_is(1);
+    }
+
+    // Succeeds to read now
+    for arg in ["-c", "--read-rc"] {
+        ts.ucmd().arg(arg).arg(pid.to_string()).succeeds();
+    }
+}
```

Comment:
I think you can run the functionality of this test in the CI only, otherwise the test will fail if there is already a `.pmaprc` in the home directory. Using `is_ci()` should do the trick.